File: src/main/java/jukeboxmc/event/player/PlayerBucketFillEvent.java
package jukeboxmc.event.player;

import org.jukeboxmc.block.Block;
import org.jukeboxmc.item.Item;
import org.jukeboxmc.player.Player;

/**
 * @author Kaooot
 * @version 1.0
 */
public class PlayerBucketFillEvent extends PlayerBucketEvent {

    public PlayerBucketFillEvent( Player player, Item bucket, Item itemInHand, Block clickedBlock ) {
        super( player, bucket, itemInHand, clickedBlock );
    }
}

File: JukeboxMC-API/src/main/kotlin/org/jukeboxmc/api/event/player/PlayerBucketFillEvent.kt
package org.jukeboxmc.api.event.player

import org.jukeboxmc.api.block.Block
import org.jukeboxmc.api.item.Item
import org.jukeboxmc.api.player.Player

class PlayerBucketFillEvent(
    player: Player,
    bucket: Item,
    itemInHand: Item,
    clickedBlock: Block
) : PlayerBucketEvent(player, bucket,
    itemInHand,
    clickedBlock
){

}